source/extensions/filters/http/ext_authz/config.cc

```diff
@@ -27,8 +27,8 @@ Server::Configuration::HttpFilterFactoryCb ExtAuthzFilterConfig::createFilter(
   return [ grpc_service = proto_config.grpc_service(), &context, filter_config,
            timeout_ms ](Http::FilterChainFactoryCallbacks & callbacks) {
     auto async_client_factory =
-        context.clusterManager().grpcAsyncClientManager().factoryForGrpcService(grpc_service,
-                                                                                context.scope());
+        context.clusterManager().grpcAsyncClientManager().factoryForGrpcService(
+            grpc_service, context.scope(), false);
```

Comment:
Do you plan on enabling this for ext_authz?

Reply:
I'm guessing you opted to cleanly split this into a followup PR. I think we can ship this after addressing the below nit.

Reply:
@htuch Yes. I figured it would be easier to reason the authz changes post gRPC changes were ok. Thanks for the fast review and will address the feedback.